app/assets/javascripts/submission.js

```diff
@@ -49,7 +49,7 @@ function initSubmissionShow(parentClass, mediaPath, token) {
 
             $(".tab-link-marker").removeClass("tab-link-marker");
             $(".feedback-table .nav-tabs > li a").filter(function () {
-                return $(this).attr("href").indexOf("#" + tab) === 0;
+                return $(this).attr("href").indexOf(tab) > 0;
```

Comment:
```suggestion
                return $(this).attr("href").indexOf(`#tab-${tab}`) === 0;
```
I think this makes more sense, since the word `code` will probably show up in other tab id's, which is the most common use of this code.